servers/zts/src/main/resources/messages/unrefreshed-certs.html

```diff
@@ -45,7 +45,14 @@
         </table>
         <hr>
         <div class="bt unrefreshedcerts">
-            <br>Please review this list and, if necessary, login to the hosts and verify sia runs on schedule.
+            <br>Please review this list and take one of the following actions:
+            <br>
+            <br> 1. For entries belonging to hosts that were re-bootstrapped or changed identity, please delete the
```

Comment:
let's change the order. some engineers will just do the first step and delete the record and make things worst. So we should say that you should login first and verify that it's working correctly first.